eng/_util/cmd/updatecryptodocs/updatecryptodocs.go

```diff
@@ -187,39 +185,48 @@ func printTable(w io.Writer, section Section) {
 
 		itemNotes := item.Notes
 		if item.MinGoVersion != "" {
-			note := fmt.Sprintf("Available starting in the Microsoft build of Go %s.", item.MinGoVersion)
```

Comment:
I think I preferred this original wording. Not going to block the PR if you disagree though.